pkg/apiserver/registry/pedia/collectionresources/rest.go

```diff
@@ -88,6 +88,10 @@ func (s *REST) Get(ctx context.Context, name string, _ *metav1.GetOptions) (runt
 	query := request.RequestQueryFrom(ctx)
 	pediascheme.ParameterCodec.DecodeParameters(query, pediav1alpha1.SchemeGroupVersion, &opts)
 
+	// collection resources don't support attach remaining count
+	// ignore opts.AttachRemainingCount
+	opts.AttachRemainingCount = nil
```

Comment:
@wzshiming  The ignore operation is placed where the option is generated, and support will be provided in the future